plugins/services/src/js/filters/__tests__/GenericObject-test.ts

```diff
@@ -16,7 +16,7 @@ describe("GenericObject", () => {
     });
     it("can filter for undefined", () => {
       expect(filterByObject({ foo: true }, { foo: undefined })).toEqual(false);
-      expect(filterByObject({ foo: true }, { bar: undefined })).toEqual(true);
+      expect(filterByObject({ foo: true }, { bar: undefined })).toEqual(false);
```

Comment:
you should just remove this line if this doesn't work anymore. This was testing that you could filter by a key *not* being present. But that feature is not required. So we should just remove this.

Reply:
i removed "can filter for undefined". please don't hesitate to make changes to my changes! :)